airflow/jobs.py

```diff
@@ -1187,7 +1187,8 @@ def _process_dags(self, dagbag, dags, tis_out):
             if dag_run:
                 expected_start_date = dag.following_schedule(dag_run.execution_date)
                 if expected_start_date:
-                    schedule_delay = dag_run.start_date - expected_start_date
+                    schedule_delay = \
+                        (dag_run.start_date - expected_start_date).total_seconds() * 1000
```

Comment:
* 1000 would make for milliseconds, not microseconds, right?

Reply:
Ah I mean milliseconds